src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -36,7 +36,8 @@ const DiscoverySearch = ({
   fetchDiscoveryData = () => {},
   fetchData,
   onCancelToken = () => {},
-  hideBackIcon = false
+  hideBackIcon = false,
+  autoFocus = true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `autoFocus` prop is added with a default value of `true`. This prop is passed to the `TextInput` component to automatically focus on it when the component mounts. However, this can sometimes lead to unwanted behavior, such as the keyboard popping up unexpectedly. It's good that you've made it configurable via props.
```diff
  onCancelToken = () => {},
  hideBackIcon = false
+ autoFocus = true
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->